notebooks/02.02-The-Basics-Of-NumPy-Arrays.ipynb

```diff
@@ -11,6 +11,24 @@
     "*The text is released under the [CC-BY-NC-ND license](https://creativecommons.org/licenses/by-nc-nd/3.0/us/legalcode), and code is released under the [MIT license](https://opensource.org/licenses/MIT). If you find this content useful, please consider supporting the work by [buying the book](http://shop.oreilly.com/product/0636920034919.do)!*"
```

Comment:
<p>this is <span class="mention" data-value="amit1rrr" data-denotation-char="@">﻿<span contenteditable="false">@amit1rrr</span>﻿</span> mentioning</p>
<br/>

 _Reply via <a href='http://127.0.0.1:8000/amit1rrr/PythonDataScienceHandbook/pull/54/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>